widget/popup_menu.go

```diff
@@ -107,7 +107,7 @@ func (p *PopUpMenu) adjustedPosition(pos fyne.Position, size fyne.Size) fyne.Pos
 // NewPopUpMenuAtPosition creates a PopUp widget populated with menu items from the passed menu structure.
 // It will automatically be positioned at the provided location and shown as an overlay on the specified canvas.
 //
-// Deprecated: Use ShowPopUpMenuAtPosition instead.
+// Deprecated: This will no longer show the pop-up in 2.0. Use ShowPopUpMenuAtPosition() instead.
```

Comment:
I'm not too sure this particular line is correct - the is an old constructor and may not be kept with different semantics. That's why it was shown as a simple use-a-different-method sort of warning

Reply:
I must say that I am very confused then. Are we supposed to keep NewPopUpMenu() then or not?

Reply:
If we are deprecating both this and NewPopUpMenu() then we have no constructors that actually returns a PopUpMenu instance.

Reply:
Those both return *PopUp... the new functions work on *PopUpMenu - we need to remove the old ones and then re-add a new one, and so what exactly will be the transition is undefined - hence why we are pushing off these.

Reply:
Ah, that explains things. Thank you. I was very confused why we were removing functions that seemed to have no real alternative :)